### ISSUE TITLE
Line occasionally misaligns when association is associated to a another association

### ISSUE THREAD

**Issue by kdrangej** [archimatetool/archi#675]
### Version of Archi
4.7.1 - build 202007151058

### Archi Plug-ins
![image](https://user-images.githubusercontent.com/3890455/93868743-32427c00-fccb-11ea-9022-5eb96f4a7c65.png)

### Operating System
Windows 10 Pro

### Expected Behaviour
Line should start always from border of the box
![image](https://user-images.githubusercontent.com/3890455/93868639-07f0be80-fccb-11ea-9534-b1703c7da759.png)

### Actual Behaviour
Line starts from center of the box, when it is placed near, in the lower right hand side of the association. Sometimes it is correct, sometimes it is incorrect. The error occurs intermittenly. 
![image](https://user-images.githubusercontent.com/3890455/93868419-b34d4380-fcca-11ea-9e53-ad164d688555.png)

### Steps to Reproduce the Behaviour
1. Add user business actor A1, A2, and business role R1,  
2  Add assignment relation REL_1 from A1 to R2. 
3. Add assignment ASS_1 from A2 to REL_1
4. Place A2 lower right hand side of diagram. Seems to be triggered when the association is between 21:00 and 23:00 as on a clock.

**Comment by Phillipus** [comment]
Yes, this can happen when one object is quite close to another using the orthogonal anchor. A workaround is to move "A2" a bit further away. 

We'll investigate.

**Comment by jbsarrodie** [comment]
Hi. This is on my side on my todo list (and has been for a long time). I'll havr a look.

**Comment by davevint** [comment]
I'm having a similar issue I think. Originally the model (created elsewhere) was imported via the Open Exchange format, but I get the same when I construct from scratch in Archi. See:
![image](https://user-images.githubusercontent.com/34168609/116429865-307eff00-a83e-11eb-9ed8-c689f0201410.png)
In this case, I can't slightly shift the interface to get rid of both overlapping lines, just one or the other. I've attached archimate file - zipped so that I can attach it.
[relationships_on_top.zip](https://github.com/archimatetool/archi/files/6392983/relationships_on_top.zip)
